exampleSite/static-main/js/index.js

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+class Editor extends React.PureComponent {
+  render() {
+    return (
+      <div className="container py-4 text-center">
+        <h3 className="text-dark">Editor Fragment</h3>
+        <h4 className="text-dark">This fragment supports React Portals. Here you are looking at a React Component.</h4>
+        <form action="#">
+          <div class="row pt-3">
+            <div class="col-md-6">
+              <div className="form-group">
+                <input type="text" className="form-control" placeholder="Enter your name" />
```

Comment:
This looks good to me.  I think this example is descriptive enough too.
Quick question how do you generate the JSON schema from the form?

Reply:
This is a POC and not really user friendly and it has one bug. The file located in `syna/static-main/js/editor.js` shows what we consider the standard editor. It's using JSON Schema and a component developed by Mozilla.